src/customMetadata/GS_Admin_Checklist_Item.Create_a_Donation.md

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>Create a Donation</label>
+    <protected>false</protected>
```

Comment:
@fparodi-salesforce I know this isn't ready for review, however I believe these records need to be marked as Protected to ensure they are "Developer Controlled". Otherwise, once deployed into an org we won't be able to update them again. It's very much the same reason why all custom labels have to be marked as Protected.
@dcaballeroSf 
https://help.salesforce.com/articleView?id=custommetadatatypes_req_managed_pkgs.htm&type=5